.github/workflows/continuous-integration.yml

```
@@ -33,7 +33,7 @@ jobs:
 
           # set up sparse checkout
           git sparse-checkout init
-          git sparse-checkout set '**/*.md' '**/.remark*' '**/*.json' '**/*.yaml' '.github/scripts'
+          git sparse-checkout set '**/*.md' '**/.remark*' '**/*.json' '**/*.yaml' '.github/scripts' 'wiki/**/*.png' 'wiki/**/*.jpg' 'wiki/**/*.gif'
```

(also it seems the news part got lost in some commit)

```suggestion
          git sparse-checkout set '**/*.md' '**/.remark*' '**/*.json' '**/*.yaml' '.github/scripts' 'wiki/**/*' 'news/**/*'
```

---

If you're looking to improve times (i don't think it's really required) you could also clone using a max depth of 1 see https://www.perforce.com/blog/vcs/git-beyond-basics-using-shallow-clones